runtime/src/snapshot_package.rs

```diff
@@ -228,18 +180,10 @@ impl SnapshotArchiveInfoGetter for SnapshotPackage {
     }
 }
 
+/// Snapshots come in two flavors, Full and Incremental.  The IncrementalSnapshot has a Slot field,
+/// which is the incremental snapshot base slot.
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum SnapshotType {
     FullSnapshot,
-    IncrementalSnapshot,
-}
-
-impl SnapshotType {
-    /// Get the string prefix of the snapshot type
-    pub fn to_prefix(&self) -> &'static str {
-        match self {
-            SnapshotType::FullSnapshot => TMP_FULL_SNAPSHOT_PREFIX,
-            SnapshotType::IncrementalSnapshot => TMP_INCREMENTAL_SNAPSHOT_PREFIX,
-        }
-    }
+    IncrementalSnapshot(Slot),
 }
```

Comment:
While looking through the actual archive and unarchive functions more, I realized that the prefix didn't need to be unique between full and incremental snapshots. So that just makes this a bit simpler.